x/warp/README.md

```diff
@@ -181,7 +154,7 @@ The Warp Precompile was designed with the intention of minimizing the trusted co
 
 The Warp Precompile itself provides ONLY the following ability:
 
-send a verified message from a caller on blockchain A to a destination address on blockchain B
+- Send a verifiable message from a caller on blockchain A to a destination address on blockchain B
```

Comment:
Line 157 and line 163 seem to contradict each other.

In line 157, we are guaranteeing that the Warp Precompile sends a message from blockchain A to B. Therefore, one can logically deduce that the message will *eventually* be delivered. However, line 163 states that the Warp Precompile does not guarantee eventual message delivery.